recipes/simpeg/meta.yaml

```diff
@@ -0,0 +1,95 @@
+package:
```

Comment:
You might consider using jinja template variables as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L1-L19

Reply:
done